locations/spiders/aldi.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+import re
+from uszipcode import ZipcodeSearchEngine
```

Comment:
Please see my comment in the PR to more efficiently query this site without including this library.